zk/src/archive/web/js/zk/widget.js

```diff
@@ -2657,7 +2657,18 @@ function () {
 	 * @return zk.Widget this widget.
 	 * @since 5.0.4
 	 */
-	rerender: function (skipper) {
+	/** @param boolean doLater is an flag to let the widget rerender later.
+	 * This parameter only works for widget call the rerender in the setAttrs phase,
+	 * avoiding widget rerendering twice or more in setAttr phase.
+	 * Widget would do rerender right after it has done all the cmds from server.

```

Comment:
1. just modify javadoc above.
2. Add a notice that calling `wgt.rerender(null, true)` outside setters will be ignored and do nothing! (wrong usage)